include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2906,25 +2733,20 @@ namespace internal
 
   template <typename Number>
   void
-  adjust_for_face_orientation(const unsigned int            dim,
-                              const unsigned int            n_components,
-                              const unsigned int            face_orientation,
-                              const Table<2, unsigned int> &orientation_map,
```

Comment:
You are merging these two at the calling function (at least twice). I need to check what ECL does in the case. In this case we have multiple face orienations.